readers-frontend/src/pages/about-us.astro

```diff
@@ -0,0 +1,45 @@
+---
+import Layout from "src/layouts/Layout.astro";
+---
+
+<Layout active={undefined} style={"nouse"} title={"About Us - Nouse"}>
+  <div class="h-[69vh] md:h-[66vh] px-[3%] pt-8 sm:px-[0.5%] 2xl:px-[13%]">
```

Comment:
The page height values seem arbitrary numbers. I'm guessing they're here so that the footer is always at the bottom? Although the footer isn't at the bottom (See white space below). 

![Screenshot from 2023-06-10 09-59-27](https://github.com/yorknouse/website/assets/24878411/5e63d13f-b5d0-4084-b8b1-150e6a72fe62)

Also, consider that vh unit doesn't really work in mobile browsers. Provide dvh variant if using this approach